packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -6,55 +6,54 @@ import { RouteComponentProps, useNavigate } from '@reach/router';
 import React, { useCallback, useState } from 'react';
 import { useForm } from 'react-hook-form';
 import { logViewEvent, usePageViewEvent } from '../../lib/metrics';
-import { useAccount, useAlertBar } from '../../models';
+import { useAccount, useFtlMsgResolver } from '../../models';
+
 import { FtlMsg } from 'fxa-react/lib/utils';
-import { useFtlMsgResolver } from '../../models/hooks';
 
 import { InputText } from '../../components/InputText';
 import CardHeader from '../../components/CardHeader';
 import WarningMessage from '../../components/WarningMessage';
 import LinkRememberPassword from '../../components/LinkRememberPassword';
+
 import { MozServices } from '../../lib/types';
 import { REACT_ENTRYPOINT } from '../../constants';
 
-// --forceAuth-- is a hint to the signup page that the user should not
-// be given the option to change their address
-// can be triggered by clicking on "Forgot password?" from /force_auth?email=user@domain.com
-// where the provided email is a registered account
-// (e.g., if coming from force_auth or a relying party)
-
-// --canGoBack-- determines if the user can navigate back to an fxa entrypoint
-
-// --serviceName-- is the relying party
+import AppLayout from '../../components/AppLayout';
+import { composeAuthUiErrorTranslationId } from '../../lib/auth-errors/auth-errors';
 
 export const viewName = 'reset-password';
 
 export type ResetPasswordProps = {
-  serviceName?: MozServices;
   prefillEmail?: string;
   forceAuth?: boolean;
-  canGoBack?: boolean;
+  serviceName?: string;
 };
 
 type FormData = {
   email: string;
 };
 
+export const routeViewName = 'reset-password';
```

Comment:
There is a `viewName` declared on line 24 - should probably just keep one of the two?